entries/n/nuri.com.json

```diff
@@ -1,12 +1,13 @@
 {
-  "Bitwala": {
-    "domain": "bitwala.com",
-    "url": "https://www.bitwala.com/",
+  "Nuri": {
+    "domain": "nuri.com",
+    "additional-domains": [
+      "bitwala.com"
+    ],
     "tfa": [
-      "sms",
```

Comment:
As @Sami32 mentions, it does appear that they also support SMS 2FA - https://support.nuri.com/hc/en-gb/articles/360009149240-TAN